bin/convert-argv.js

```diff
@@ -86,7 +86,9 @@ module.exports = function(yargs, argv, convertOptions) {
 							registerCompiler(moduleDescriptor[i]);
 							break;
 						} catch(e) {
-							// do nothing
+							if(i === moduleDescriptor.length - 1 && e.message.match(/Cannot find module/)) {
+								throw new Error("Cannot resolve config file. Install appropriate compiler and try again.");
```

Comment:
Since `js-interpret` package provides multiple options (some are already deprecated and inappropriate) for each file type, it might be also misleading. I actually thought about giving a link to [documentation page](https://webpack.js.org/configuration/configuration-languages/).